src/src/Commands/GetCommand.php

```diff
@@ -142,13 +157,12 @@ protected function execute( InputInterface $input, OutputInterface $output ): in
 		}
 
 		$table = new Table( $output );
-		$table->setColumnMaxWidth( 1, 80 );
 		$table
 			->setHorizontal()
 			->setStyle( 'compact' )
 			->setHeaders( array_keys( $test_run ) )
 			->setRows( [ $test_run ] );
-		$table->render();
+	$table->render();
```

Comment:
Not sure if this was intention but looks like a tab was removed here

Reply:
:+1: PHPCBF to the rescue